mgmtsystem_action/reports/mgmtsystem_action_report.py

```diff
@@ -37,10 +37,9 @@ class MgmtsystemtActionReport(models.Model):
     def _query(
         self, with_clause="", fields="", where_clause="", groupby="", from_clause=""
     ):
-        with_ = ("WITH %s" % with_clause) if with_clause else ""
+        with_ = ("WITH %s", with_clause) if with_clause else ""
```

Comment:
wrong logic syntax, compared 2 results of old and new, they should be the same.
![image](https://github.com/user-attachments/assets/3dc7ac0d-7acf-4983-b4e1-5c81c7dcb4a2)